src/stan/variational/advi.hpp

```diff
@@ -334,6 +520,14 @@ namespace stan {
 
         // draw more samples from posterior and write on subsequent lines
         if (out_stream_) {
```

Comment:
i don't think so. we need to check for `out_stream_` otherwise there's no csv file to write posterior samples to. we need to check for `print_stream_` before spitting things out to the terminal.